webapp/components/Modal/ReportModal.vue

```diff
@@ -26,8 +48,10 @@
 </template>
 
 <script>
-import gql from 'graphql-tag'
 import { SweetalertIcon } from 'vue-sweetalert-icons'
+import { reportMutation } from '~/graphql/Moderation.js'
+import { valuesReasonCategoryOptions } from '~/constants/modals.js'
+import validReport from '~/components/utils/ReportModal'
 
 export default {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-14T19:40:00Z" title="Monday, October 14th 2019, 9:40:00 pm +02:00">Oct 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -40,10 +62,55 @@ export default {
     id: { type: String, required: true },
   },
   data() {
+    let valuesReasonCategoryOptions = [
```


Put that into global constants?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-15T17:58:32Z" title="Tuesday, October 15th 2019, 7:58:32 pm +02:00">Oct 15, 2019</time>_

---


done